### PR TITLE
add arkh logo

### PR DESCRIPTION
**Pull request by kynraze**

### CODE REVIEW

File: arkh/assetlist.json [hunk 1]
```diff
@@ -5,21 +5,17 @@
...
-            "denom": "uarkh",
-            "exponent": 0
-          },
           {
             "denom": "arkh",
             "exponent": 6
           }
         ],
-        "base": "uarkh",
+        "base": "arkh",
```
Comment (JeremyParish69) [inline arkh/assetlist.json:13]: We need to keep some denom unit of exp 0, and base must point to that.

File: arkh/chain.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   "fees": {
     "fee_tokens": [
       {
-        "denom": "uarkh",
+        "denom": "arkh",
```
Comment (JeremyParish69) [inline arkh/chain.json:19]: this should go back to uarkh as well

File: arkh/chain.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "chain_name": "arkh",
   "status": "live",
   "network_type": "mainnet",
-  "website": "https://arkh.com/",
+  "website": "http://arkhadian.com/",
```
Comment (JeremyParish69) [inline arkh/chain.json:6]: Site doesn't work? It the domain reserved?

Comment (kynraze) [reply arkh/chain.json:6]: nah, the site is currently down But the previous site has no relation to this chain

**Pull request merged**